### DISCUSSION TITLE
목차에서 두 번 뜨는 현상

### DISCUSSION THREAD

**Discussion by damon-911** [pronist/hELLO#188] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/79182122/216758837-7ec94e05-3bda-42f2-b649-5ec25cc1bccf.png)
**각각 제목1, 제목2 로 입력하였는데 글 작성을 완료하면 밑의 사진처럼 나타납니다. 본문이 두번 작성되는 것 같습니다
해결방법이 혹시 있을지 문의드립니다.**
![image](https://user-images.githubusercontent.com/79182122/216758851-df32e67f-b03c-4aa6-9968-95af6d1fab03.png)

**Comment by damon-911** [comment, marked as answer]
코드블록에 Line Number를 추가하는 코드를 추가해서 생기는 문제 같습니다.
해당 코드를 제외하면 정상 작동하는 것을 확인했습니다.
```
<script defer src='//cdn.jsdelivr.net/npm/highlightjs-line-numbers.js@2.8.0/dist/highlightjs-line-numbers.min.js'></script>
<script>
      window.addEventListener('DOMContentLoaded', () => {
        Alpine.start()
    
        hljs.initLineNumbersOnLoad()
      })
</script>
```

**Reply by pronist** [reply]
**script.js** 에서 `Alpine.start()` 를 기본으로 호출합니다. 두 번 호출하셨기 때문에 발생한 문제입니다.

**Reply by damon-911** [reply]
그럼 해당 코드는 어떻게 작성해야 할까요?
script.js에 저 코드를 추가해야 되는 걸까요?

**Reply by pronist** [reply]
script.js 에 있는 코드를 지우고 skin.html 에 추가하시면 됩니다.

**Reply by damon-911** [reply]
해결했습니다. 감사합니다!
